packages/components/src/components/domainproperties/dataset/DatasetDesignerPanels.tsx

```diff
@@ -73,10 +214,34 @@ export class DatasetDesignerPanelImpl extends React.PureComponent<Props & Inject
                         onTogglePanel(0, collapsed, callback);
                     }}
                     onChange={this.onPropertiesChange}
-                    showDataspace={showDataSpace}
-                    showVisitDate={showVisitDate}
                 />
-            </>
+                <DomainForm
+                    key={model.domain.domainId || 0}
+                    domainIndex={0}
+                    domain={model.domain}
+                    headerTitle="Fields"
+                    helpNoun="dataset"
+                    helpTopic={null} // null so that we don't show the "learn more about this tool" link for this domains
+                    onChange={this.onDomainChange}
+                    setFileImportData={this.setFileImportData}
+                    controlledCollapse={true}
+                    initCollapsed={currentPanelIndex !== 1}
+                    validate={validatePanel === 1}
+                    panelStatus={
+                        model.isNew()
+                            ? getDomainPanelStatus(1, currentPanelIndex, visitedPanels, firstState)
+                            : 'COMPLETE'
+                    }
+                    showInferFromFile={true}
+                    containerTop={containerTop}
+                    onToggle={(collapsed, callback) => {
+                        onTogglePanel(1, collapsed, callback);
+                    }}
+                    useTheme={useTheme}
+                    renderDatasetColumnMapping={this.datasetColumnMapping}
+                    successBsStyle={successBsStyle}
+                />
+            </BaseDomainDesigner>
```

Comment:
See ListDesignerPanels.tsx, I think we'll want to add the same type of `<Progress>` component usages to show a progress bar in case there is a large file import taking place.